pages/accountLists/[accountListId]/tasks/[[...contactId]].page.tsx

```diff
@@ -406,8 +406,8 @@ const TasksPage: React.FC = () => {
                             onTaskCheckToggle={toggleSelectionById}
                             isChecked={isRowChecked(task.id)}
                             useTopMargin={index === 0}
-                            contactDetailsOpen={contactDetailsOpen}
                             removeSelectedIds={deselectMultipleIds}
+                            filterPanelOpen={filterPanelOpen}
```

Comment:
I think that this change is making the task rows always be in condensed mode, even with a large browser window and without having the contact details open.

Reply:
Yes, that change was intentional. It seems like the condensed view is also useful for small or medium screens. Especially if we're trying to display the same amount of data that the old UI did.

I was thinking we could just leave the condensed view for all screen sizes. But what do you think? Should we show the non-condensed view for extra large screen sizes?

Reply:
I've asked Scott for his feedback.

Reply:
I asked Scott and attached is his reply. 
I think the test account tends to have shorter task titles so it seems empty. But on my personal, the automated tasks that the API makes have very long titles and it is nice to have the room on larger screens.

<img width="521" alt="Screenshot 2024-10-17 at 12 34 14 PM" src="https://github.com/user-attachments/assets/5de05186-4c73-4dca-ad11-c640e7ef832f">

Reply:
Sounds good!